README.md

```diff
@@ -0,0 +1,125 @@
+
+## Installation
+
+To install the module, use the command line to run this command in an Apostrophe project's root directory:
+
+```
+npm install @apostrophecms/sync-content
+```
+
+## ⚠️ Warnings
+
+This tool makes big changes to your database. There are no confirmation prompts in the current command line interface. Syncing "from" staging or production to your local development environment is generally safe, but take care to think about what you are doing.
+
+## Usage
+
+Configure the `@apostrophecms/sync-content` module in the `app.js` file:
+
+```javascript
+require('apostrophe')({
+  shortName: 'my-project',
+  modules: {
+    '@apostrophecms/sync-content': {
+      // Our API key, for incoming sync requests
+      apiKey: 'choose-a-very-secure-random-key',
+      environments: {
+        staging: {
+          label: 'Staging',
+          url: 'https://mysite.staging.mycompany.com',
+          // Their API key, for outgoing sync requests
+          apiKey: 'choose-a-very-secure-random-key'  
+        }
+      }
+    }
+  }
+});
+```
+
+Note that the command line interface can also 
+
+### ✅ Syncing via command line tasks
+
+#### ✅ Syncing from another site
+
+```bash
+# ✅ sync all content from another environment
+node app @apostrophecms/sync-content:sync --from=staging
+
+# Pass a site's base URL and api key directly, bypassing `environments`
+node app @apostrophecms/sync-content:sync --from=https://site.com --api-key=xyz
+
+# ✅ sync content of one piece type only, plus any related
+# documents. If other content already exists locally, purge it
+node app @apostrophecms/sync-content:sync --from=staging --type=article
+
+# ✅ Same, but keep existing content of this type too
+node app @apostrophecms/sync-content:sync --from=staging --type=article --keep
+
+# ✅ sync content of one piece type only, without related documents
+node app @apostrophecms/sync-content:sync --from=staging --type=article --related=false
+
+# ✅ sync content of one piece type only, matching a query
+node app @apostrophecms/sync-content:sync --from=staging --type=article --query=tags[]=blue
+
+# 🚧 skip media, for speed (you will see broken images)
+node app @apostrophecms/sync-content:sync --from=staging --skip-media
+
+# 🚧 sync all content of this site TO another environment.
+# If other content already exists in the OTHER environment, purge it
+node app @apostrophecms/sync-content:sync --to=staging
+```
+
+* ✅ You must specify `--from` to specify the environment to sync with, as seen in your configuration above, where `staging` is an example. You can also specify the base URL of the other environment directly for `--from`, in which case you must pass `--api-key` as well.
+* 🚧 Later `--to` will also be supported. 
+* ✅ You may specify `--type=typename` to specify one content type only. This must be a piece type, and must match the `name` option of the type (**not** the module name, unless they are the same).
```

Comment:
If you're considering this an alpha state for the docs, that might be alright. But it's not a super clear way to present this information. Instead I'd have an options section that covers each one by one with subheads or something rather than a bulleted list that's hard to scan.